.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think we don't need to add `.idea` directory.

Reply:
Fixed in [403399d](https://github.com/medibloc/panacea-dep-consumer/pull/1/commits/403399d7e78792bb7722ff7d52f52c37c72ae1b1).